### DISCUSSION TITLE
Remove AxxSolder_firmware/Debug directory from the repository

### DISCUSSION THREAD

**Discussion by v-barshaw** [AxxAxx/AxxSolder#54] [Build related questions]
The size of AxxSolder_firmware directory is 60MB, of which 47MB are AxxSolder_firmware/Debug directory.

All files in AxxSolder_firmware/Debug are generated. Removing the directory from the repository and adding it to .gitignore file would save network bandwidth and disk space.

**Comment by AxxAxx** [comment]
Good input!

I'll fix that.

Thanks.
